src/components/Farmhand/Farmhand.js

```diff
@@ -314,12 +315,7 @@ export default class Farmhand extends Component {
   get playerInventoryQuantities() {
     const { inventory } = this.state
 
-    return itemIds.reduce((acc, itemId) => {
-      const itemInInventory = inventory.find(({ id }) => id === itemId)
-      acc[itemId] = itemInInventory ? itemInInventory.quantity : 0
-
-      return acc
-    }, {})
```

Comment:
This getter's logic was moved to an external helper so that it can be tested in isolation.

Reply:
it looks like the getter is only used in one place; what do you think about just calling the helper there and removing the getter?